skimage/transform/radon_transform.py

```diff
@@ -13,16 +13,22 @@
 __all__ = ['radon', 'order_angles_golden_ratio', 'iradon', 'iradon_sart']
 
 
-def radon(image, theta=None, circle=True, *, preserve_range=False):
+def _get_default_center(image_shape, legacy: bool = True):
+    if legacy:
+        return image_shape // 2
+    else:
+        return (image_shape - 1) / 2
+
+
+def radon(image, theta=None, circle=True, *, preserve_range=False, center=None):
     """
     Calculates the radon transform of an image given specified
```

Comment:
```suggestion
    """Calculate the Radon transform of an image given the
```
Doesn't come from this PR, but we might as well fix this minor issue.